webapp/src/pages/FoodCheckout.jsx

```diff
@@ -419,6 +423,11 @@ function OrderSummary({ checkout, chosenInstrument }) {
             />
           </>
         )}
+        {memberUnverified && (
+          <Alert variant="danger" className="mt-3">
+            {readOnlyReason(user, "read_only_unverified")}
+          </Alert>
+        )}
         {checkout.checkoutProhibitedReason === "charging_prohibited" && (
```

Comment:
This condition no longer exists (no more 'charging prohibited' condition on the offering), I forgot to remove it when I overhauled charging.